README.md

```diff
@@ -19,6 +19,15 @@ Marquez is an open source **metadata service** for the **collection**, **aggrega
 
 Marquez is an [LF AI & Data Foundation](https://lfaidata.foundation/projects/marquez) incubation project under active development and we'd love your help!
 
+## Code of Conduct
+We are proud to be an adopter of [Contributor Covenant](https://www.contributor-covenant.org/).
+_Pledge your respect and appreciation for contributors of all kinds to your open source project._
+
+There are 184 two-letter language codes ([ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1)) registered as of December 2018.
+We’re always looking for new localizations of Contributor Covenant and are thankful to the volunteers who spend their time on translations.
+
+Instructions for translations can be found [here](https://github.com/ContributorCovenant/contributor_covenant#translating).
```

Comment:
I don't think this section is relevant to the Marquez README.
Having the badge and CODE_OF_CONDUCT.md at the root of the repo is sufficient.